### PR TITLE
Clean up some alerts detected by LGTM [pypi]

### PR DESCRIPTION
**Pull request by paulmelnikow**
https://lgtm.com/projects/g/badges/shields/alerts/?mode=list

To help us keep up to date on this, We could consider adding the readme badge and/or the GitHub integration.

### CODE REVIEW

**Comment by tooomm** [comment]
Which `WIP` App did you add to the repo?
It seems to be broken on PR titles which contain squared brackets. I happily report this if you let me know which service it is @paulmelnikow.

**Comment by paulmelnikow** [comment]
It's this one: https://github.com/apps/wip

Yea, I've noticed it sometimes shows the wrong status.

File: services/pypi/pypi.tester.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const isPsycopg2Version = Joi.string().regex(/^v([0-9][.]?)+$/)
 
 // These regexes are the same, but declared separately for clarity.
 const isPipeSeparatedPythonVersions = Joi.string().regex(
-  /^([0-9]+.[0-9]+(?: \| )?)+$/
+  /^([0-9]+\.[0-9]+(?: \| )?)+$/
```
Comment (PyvesB) [inline services/pypi/pypi.tester.js]: Nice catch!

Comment (paulmelnikow) [reply services/pypi/pypi.tester.js]: It's a really helpful service!

**Comment by paulmelnikow** [comment]
@samlanning I'm curious to see if this change fixed the alerts. Is there a way I can trigger https://lgtm.com/projects/g/badges/shields/ to update using the latest build?

**Comment by samlanning** [comment]
@paulmelnikow alerts updated: https://lgtm.com/projects/g/badges/shields/alerts/?mode=list

**Comment by paulmelnikow** [comment]
Looks like I got 3/4 right!

**Comment by samlanning** [comment]
@paulmelnikow in fairness, the message for the one you didn't fix did change, it's still triggered by the same query, but it's pointing out something different now :)

**Pull request merged**